src/System.Configuration.ConfigurationManager/tests/System.Configuration.ConfigurationManager.Tests.csproj

```diff
@@ -4,6 +4,8 @@
   <PropertyGroup>
     <ProjectGuid>{7669C397-C21C-4C08-83F1-BE6691911E88}</ProjectGuid>
     <TestCategories>InnerLoop;OuterLoop</TestCategories>
+    <!-- XUnit AppDomain has different behavior when accessing AppDomain.BaseDirectory (trailing/no trailing slash) -->
```

Comment:
I would maybe be more detailed on the comment above saying that when Xunit create AppDomain it creates it with no trailing slash on its path.